docs/configuration.md

```diff
@@ -2018,6 +2018,50 @@ Apart from these, the following properties are also available, and may be useful
 </tr>
 </table>
 
+### Barrier Execution Mode
+
+<table class="table">
+<tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
+<tr>
+  <td><code>spark.barrier.sync.timeout</code></td>
+  <td>365d</td>
+  <td>
+    The timeout in seconds for each <code>barrier()</code> call from a barrier task. If the
+    coordinator didn't receive all the sync messages from barrier tasks within the
+    configed time, throw a SparkException to fail all the tasks. The default value is set
```

Comment:
nit: `configed` -> `configured`

Reply:
Thanks! Actually, this description is from [here](https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/internal/config/package.scala#L1061). So it might be better to modify it too.

Reply:
Yes, I think so. This is because other sections use `configured`.